tests/tck/features/match/SeekById.feature

```diff
@@ -252,42 +263,81 @@ Feature: Match seek by id
       """
     Then a ExecutionError should be raised at runtime: Scan vertices or edges need to specify a limit number, or limit number can not push down.
 
-  @skip
-  Scenario: test OR logic (reason = "or logic optimization error")
+  Scenario: test OR logic
```

Comment:
Add case like `match (v) where id(v) IN ['James Harden', 'Jonathon Simmons', 'Klay Thompson', 'Dejounte Murray'] OR id(v) == 'James Harden'`

Reply:
done